internal/host/validator.go

```diff
@@ -795,3 +795,21 @@ func (v *validator) printSufficientPacketLossRequirementForRole(c *validationCon
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) isDomainNameResolved(c *validationContext) ValidationStatus {
+	if c.host.DomainNameResolution == c.cluster.APIVip {
+		return ValidationSuccess
+	}
+	return ValidationFailure
+}
+
+func (v *validator) printIsDomainNameResolved(c *validationContext, status ValidationStatus) string {
+	switch status {
+	case ValidationSuccess:
+		return "API name resolution was secsessful"
+	case ValidationFailure:
+		return "Faild to resolve the API name"
```

Comment:
This error is shown to the user - I think a more verbose message is important here. Something along the lines of:

> There was an attempt to resolve the %s domain on the host. Expected domain to resolve to IP address %s but instead <the domain was not resolved/the domain was resolved to %s>. The installation will not be able to complete as long as the host DNS server is able to resolve this domain. Please create the necessary DNS entries"

Maybe @avishayt would like to have the final word on this

Reply:
Done